talent/job-descriptions/applications.md

```diff
@@ -0,0 +1,68 @@
+- At source{d} we are building the stack for the next generation of machine-learning powered developer tools.
+
+- source{d} raised over $8MM till date and we are currently growing our team. 
```

Comment:
We have raised over eight million USD so far, and we are currently growing our team.

(Avoid starting with `source{d}` as it looks weird to start with a lower case letter, and avoid ambiguous or not broadly known abbreviations such as MM)